xml/Microsoft.VisualBasic/FileSystem.xml

```diff
@@ -3660,10 +3660,7 @@
   
  </description></item><item><term><see langword="Alias" /></term><description><see langword="vbAlias" /></description><description> File has a different name.  
   
- </description></item></list>
- 
- <block subset="none" type="note">
- <para>  
+ </description></item></list><block subset="none" type="note"><para>  
```

Comment:
@daxianji007 it seems that the changes you've made in PR #1218 are controlled by the CI process, so it will be overridden by mdoc every time we run /cc @joelmartinez @dend